Bio/NeuralNetwork/Gene/Schema.py

```diff
@@ -606,7 +607,8 @@ def from_motifs(self, motif_repository, motif_percent, num_ambiguous):
         # of motifs
         total_count = self._get_num_motifs(motif_repository, all_motifs)
         matched_count = 0
-        assert total_count > 0, "Expected to have motifs to match"
+        if total_count == 0:
```

Comment:
Style wise does this seem equivalent?  ``if not total_count:``

Reply:
Yes they are equivalent.